deny.toml

```diff
@@ -39,6 +39,7 @@ allow = [
     "Apache-2.0",
     "BSD-2-Clause",
     "BSL-1.0",
+    "ISC",
```

Comment:
Thanks for adding this. We haven't yet documented this, but could you please also add `ISC` to `brush-shell/about.toml`? We recently added usage of `cargo about` to generate license notices that we can bundle with our prebuilt binaries.